README.md

```diff
@@ -1,2 +1,87 @@
-# gcxchange
-A forex exchange site
+### Gcxchange
+
+## Description 🏗️
+
+This is a website that displays exchange rates from United State Dollars to other currencies.
+
+||             Gcxchange ||
+|-|-----------------------------------|-|
+||![screenshot](./src/img/../images/gcexchange.gif)||
+
+>Live Demo:
+
+[![Netlify Status](https://api.netlify.com/api/v1/badges/2cb2eecf-7202-4f70-959f-ce20250d3d81/deploy-status)](https://[app.netlify.com/sites/gcxchange/deploys](https://gcxchange.netlify.app/))
```

Comment:
- [ ] When I click on the Netflify badge which contains the live demo, I see a blank page but when I copy the deployed link `https://gcxchange.netlify.app/` and paste it on the browser, the pages display properly. I suggest you separate the Netlify badge from the deployed link or try fixing the path so that when a user clicks on the badge the pages are displayed properly.

See the blank page:
<img width="403" alt="chrome_mcI87bP0B3" src="https://user-images.githubusercontent.com/81468675/158148150-a0663445-5ada-46ec-b442-48f7d44a19f1.png">